assets/js/base/components/cart-checkout/totals/footer-item/index.js

```diff
@@ -37,34 +38,46 @@ const TotalsFooterItem = ( { currency, values } ) => {
 
 	const parsedTaxValue = parseInt( totalTax, 10 );
 
+	const TaxLabel = ( SHOW_TAXES && parsedTaxValue !== 0 && (
+		<p className="wc-block-components-totals-footer-item-tax">
+			{ createInterpolateElement(
+				__(
+					'Including <TaxAmount/> in taxes',
+					'woo-gutenberg-products-block'
+				),
+				{
+					TaxAmount: (
+						<FormattedMonetaryAmount
+							className="wc-block-components-totals-footer-item-tax-value"
+							currency={ currency }
+							value={ parsedTaxValue }
+						/>
+					),
+				}
+			) }
+		</p>
+	) ) || <></>;
+
+	const filteredTotalDescription = createInterpolateElement(
+		__experimentalApplyCheckoutFilter( {
+			filterName: 'totalDescription',
+			defaultValue: __( '<TaxLabel/>', 'woo-gutenberg-products-block' ),
+			extensions: cart.extensions,
+			arg: { cart },
+			validation: ( value ) => mustContain( value, '<TaxLabel/>' ),
+		} ),
+		{
+			TaxLabel,
+		}
+	);
+
 	return (
 		<TotalsItem
 			className="wc-block-components-totals-footer-item"
 			currency={ currency }
 			label={ label }
 			value={ parseInt( totalPrice, 10 ) }
-			description={
```

Comment:
It looks like the description field was used only for displaying the taxes. What comest to mind is to separate the two section.
- rename the old description container (.wc-block-components-totals-item__description) to a taxes container
- add a separe section where the content comes from the filter you added. It could as well keep the name .wc-block-components-totals-item__description.

I think we should double check with design how the two sections should look.

Reply:
I've added some text in the DOM where the description wrapper currently is and there is also a visual issue for mobile, so we should definatelly bring in some designer input to this.
<img width="269" alt="Screenshot 2021-11-16 at 14 30 19" src="https://user-images.githubusercontent.com/1628454/141994499-02b84cd6-dec7-498e-8575-202c3fb3e37f.png">

Reply:
I missed Raluca's reply here while I was typing up my own comment. Her proposal sounds pretty good too. I like the idea of _explicitly_ calling out an area reserved for taxes and leaving any other descriptions to a separate area.